docs/google/devices.md

```diff
@@ -0,0 +1,194 @@
+# Google Devices
+
+This section describes the devices in Cirq for Google hardware devices and their usage.
+Since quantum hardware is an active area of research, hardware specifications and best
+practices are constantly evolving in an attempt to continuously improve performance.
+While this information should be a solid base for beginning your quantum application,
+please work with your Google sponsor to obtain the latest information on devices that
```

Comment:
is "Google sponsor" the terminology we're using?

Reply:
I think so?  We'll probably need to revamp this as we determine the process a bit better.  Did you have a suggestion on how to change this, or do you think it's okay to leave as it for now?